how-to-go-home-early-github.md

```diff
@@ -0,0 +1,75 @@
+
+### Auto label your pull requests
+
+With the new GitHub actions you can automatically label your pull requests based on what files you touched in your PR. This can help a lot with the discoverability of your pull request overview.
+
+![Alt screenshot of GitHub action on timeline](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/labeler-timeline.png)
+
+To set this up, go to your GitHub repo with actions enabled, click on "New workflow" and once in the marketplace scroll down to "Automate every step in your process" where you'll find the "labeler" action:
+
+![Alt screenshot of GitHub action in marketplace](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/labeler-marketplace.png)
+
+### Automatically request reviews with "Code owners" 
+
+We can take the automatic labeling of your PR one step further. There is a feature in GitHub called "Code owners" which let's you assign co-workers to certain parts of your application(s) GitHub will use codeowners to automatically asign reviews.
+
+![Alt screenshot of reviewers being assigned](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/codeowners.png)
+
+Setting up code owners is easy, check [this guide](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) to get started with code owners.
+
+### Let a panda spam your co-workers into submission for reviews.
+
+One thing that helped our team a lot was the addition of [Pull panda](https://pullreminders.com/). It is a Slack integration that automatically sends a message to your co-workers when someone requests their review. 
+
+When you request a review, the reviewer gets a message like:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-assigned.png)
+
+If someone approves your pull request, you'll be notified as well:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-approved.png)
+
+Even if someone comments on your pull request, panda will show you the comments right away in Slack:
+
+![Alt screenshot of pull panda request message](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/panda-comments.png)
+
+### Automatically merge the pull request
+
+Our pull request got labeled automatically, the correct reviewers got assigned automatically and they got a Slack message reminding them that there is work to be done. Now there is one last thing ~~we~~ the robots should do, merge the pull request into the master branch! If your team uses any form of automated checks on pull requests, you have probably already wasted some time blindly staring at your screen until the merge button turns green.
```

Comment:
Same here. I would suggest:
```suggestion
Our pull requests get labeled automatically, the correct reviewers get assigned automatically and they get a Slack message reminding them that there is work to be done. Now there is one last step left over for ~~us~~ the robots to do, merge the pull request into the master branch! If your team uses any form of automated checks on pull requests, you have probably already wasted some time blindly staring at your screen until the merge button turns green.
```

Reply:
The past tense is deliberate here. I am trying to build a narrative where the user creates a "virtual pull request". But maybe I can reword this a bit.